packages/expo-device/src/Device.ts

```diff
@@ -0,0 +1,87 @@
+import ExpoDevice from './ExpoDevice';
+
+import {
+  devicesWithNotch,
+  deviceNamesByCode,
+} from './DeviceConstants';
+
+import { Platform, UnavailabilityError } from '@unimodules/core';
+
+export const brand = ExpoDevice.brand;
```

Comment:
Writing exports this way if, by any chance, this code would end up being run in an application that wouldn't have `ExpoDevice` available, the application would crash immediately. Add `ExpoDevice ? ExpoDevice.brand : null`, if you want to export these constants from this file

Reply:
awesome, thanks